Header header

uint16 rx1_status
uint8  rx1_mode_of_operation
uint32 rx1_position
uint32 rx1_digital_inputs

uint16 rx2_status
uint8  rx2_mode_of_operation
uint32 rx2_velocity
uint32 rx2_digital_inputs

